src/molecules/gv-card-full/gv-card-full.js

```diff
@@ -200,12 +214,10 @@ export class GvCardFull extends ItemResource(LitElement) {
         </div>
         <div class="version"><span class="${classMap({ skeleton: this._skeleton })}">${getVersion(this._item)}</span></div>
       </div>
-      <div class="${classMap({ skeleton: this._skeleton, description: true })}">
-        ${truncate(
-          this._error ? i18n('gv-card-full.error') : this._empty ? i18n('gv-card-full.empty') : getDescription(this._item),
-          this.limit,
-        )}
-      </div>
+      <div
+        class="${classMap({ skeleton: this._skeleton, description: true })}"
+        .innerHTML="${truncate(this._getDescription(), this.limit)}"
```

Comment:
whats happen it the truncate are in middle  of <h1>... ? 🙈

Reply:
good point 🙈 weirdly it does not break

<img width="1394" alt="Screenshot 2024-02-27 at 09 58 51" src="https://github.com/gravitee-io/gravitee-ui-components/assets/3619261/49be1ad0-21c7-44df-bb47-cf383ccd0551">

Reply:
```javascript
const description =
  '<h1>Temporequo primis auspiciis in mundanum <a href="">fulgorem surgeret</a> victura dum erunt homines Roma, ' +
  'ut augeretur sublimibus incrementis, foedere pacis aeternae Virtus convenit atque  plerumque dissidentes,</h1>';
```